plasmapy/physics/parameters.py

```diff
@@ -596,139 +596,65 @@ def gyroradius(B, *args, Vperp=None, T_i=None, ion='e'):
 
     if Vperp is not None:
-        _check_quantity(Vperp, 'Vperp', 'ion_gyroradius', units.m/units.s)
+        _check_quantity(Vperp, 'Vperp', 'gyroradius', units.m/units.s)
     elif T_i is not None:
-        _check_quantity(T_i, 'T_i', 'ion_gyroradius', units.K)
-        Vperp = thermal_speed(T_i, ion=ion)
+        _check_quantity(T_i, 'T_i', 'gyroradius', units.K)
+        Vperp = thermal_speed(T_i, particle=particle)
 
-    omega_ci = gyrofrequency(B, ion)
+    omega_ci = gyrofrequency(B, particle)
 
     r_Li = np.abs(Vperp)/omega_ci
 
     return r_Li.to(units.m, equivalencies=units.dimensionless_angles())
 
 
 @check_quantity({
-    'n_e': {'units': units.m**-3, 'can_be_negative': False}
-})
-def electron_plasma_frequency(n_e):
-    r"""Calculates the electron plasma frequency.
-
-    Parameters
-    ----------
-    n_e: Quantity
-        Electron number density
-
-    Returns
-    -------
-    omega_pe: Quantity
-        Electron plasma frequency in radians per second
-
-    Raises
-    ------
-    TypeError
-        If n_e is not a Quantity
-
-    UnitConversionError
-        If n_e is in incorrect units
-
-    ValueError
-        If n_e contains invalid values
-
-    UserWarning
-        If units are not provided and SI units are assumed
-
-    Notes
-    -----
-    In a simple one dimensional model, the separation of charge
-    within a plasma creates an electric field proportional to
-    separation distance with magnitude:
-
-    .. math::
-    E = \frac{e}{\epsilon_0} n_e x
-
-    Where x is the separation distance.
-
-    The electrons will move under the action of this field with force
-    magnitude:
-
-    .. math::
-    F = e E = \frac{e^2}{\epsilon_0} n x = m_e \frac{d^2 x}{d t^2}
-
-    This is a simple harmonic oscillator. Computing its eigenfrequency
-    yields the electron plasma frequency, which is given by:
-
-    .. math::
-    \omega_{pe} = e \sqrt{\frac{n_e}{\epsilon_0 m_e}}
-
-    At present, astropy.units does not allow direct conversions from
-    radians/second for angular frequency to 1/second or Hz for
-    frequency.  The dimensionless_angles equivalency allows that
-    conversion, but does not account for the factor of 2*pi.  The
-    alternatives are to convert to cycle/second or to do the
-    conversion manually, as shown in the examples.
-
-    Example
-    -------
-    >>> from astropy import units as u
-    >>> from plasmapy import electron_plasma_frequency
-    >>> electron_plasma_frequency(1e19*u.m**-3)
-    <Quantity 178398636622.99567 rad / s>
-
-    """
-
-    omega_pe = (units.rad*e*np.sqrt(n_e/(eps0*m_e))).to(units.rad/units.s)
-
-    return omega_pe
-
-
-@check_quantity({
-    'n_i': {'units': units.m**-3, 'can_be_negative': False}
+    'n': {'units': units.m**-3, 'can_be_negative': False}
 })
-def ion_plasma_frequency(n_i, ion='p'):
-    r"""Calculates the ion plasma frequency.
+def plasma_frequency(n, particle='e'):
+    r"""Calculates the particle plasma frequency.
```

Comment:
"Defaults to the fastest, electron plasma frequency.". How's that sound?

Reply:
I don't understand what you mean.

Reply:
I mean we could change the beginning of this docstring like this:

    """Calculates the particle plasma frequency. Defaults to the fastest, electron plasma frequency. 

The default behavior of this function seems important enough to mention it straight away :)

Reply:
Ahh, Ok, sounds fine. Should something similar be done for every other function I modified?